src/geopm_prof.h

```diff
@@ -40,6 +40,13 @@
 extern "C" {
 #endif
 
+/// @brief Enum for controller launch methods
+enum geopm_ctl_e {
+    GEOPM_CTL_NONE,
+    GEOPM_CTL_PROCESS,
+    GEOPM_CTL_PTHREAD,
+};
```

Comment:
Not to stop this review, but something to consider maybe for a future change:

Does this enum really belong in geopm_prof.h? After looking at places in the code where it's used, iIt doesn't seem to be related to the profiling functions/functionality.

Alternatively, maybe geopm_prof.h might need to be renamed in the future.